src/components/PurchaseOrderDetail.vue

```diff
@@ -185,4 +185,8 @@ export default defineComponent({
   h1 {
     margin-left: 15px;
   }
+
+  ion-chip{
```

Comment:
```suggestion
  ion-chip {
```

Reply:
These selectors are too generic. They will cause problems later if we do try and add chips that have clickable functions later. Think of a different way to approach this.